src/services/importStudents/ImportStudentsContext.ts

```diff
@@ -0,0 +1,40 @@
+import {StudentsRatingWithEmail} from "../../entities/types/studentsRating";
+import {User} from "../../entities/User/User.entity";
+import {myDataSource} from "../../../config/database.configuration";
+import { StudentsRating } from "../../entities/studentsRating/studentsRating.entity";
+import {Roles} from "../../entities/types/Roles";
+import {randomUUID} from "crypto";
+
+export class ImportStudentsToDB {
+
+    public importToDb = async (studentsRating: StudentsRatingWithEmail[]): Promise<boolean> => {
+        for ( const studentData of studentsRating ) {
+            if(await this.checkUserExist(studentData)) continue;
+            const user = new User();
+            user.email = studentData.email;
+            user.password = randomUUID();
+            user.token = randomUUID();
+            user.role = Roles.STUDENT;
+            user.registerToken = randomUUID();
+
```

Comment:
dla registerToken jeżeli masz już widoczną tą część kodu to utilsach jest plik 'createRegisterToken.ts' generujący token rejestracyjny na 7 dni.

Reply:
Changes applied